leehyowonzero/3week/49995.py

```diff
@@ -0,0 +1,18 @@
+def solution(cookie):
+    answer = 0
+    for pivot in range(1,len(cookie)):
+        left = 0
+        right = len(cookie)-1
+        leftsum = sum(cookie[left:pivot])
+        rightsum = sum(cookie[pivot:right+1])
```

Comment:
```suggestion
        leftsum = sum(cookie[:pivot])
        rightsum = sum(cookie[pivot:])
```

시작과 끝은 굳이 명시해주지 않아도 됩니당~